feature-group-definitions/constraint-validation.yml

```
@@ -1,4 +1,5 @@
 name: Constraint validation API
+description: The `checkValidity()`, `reportValidity()` and `setCustomValidity()` methods are a programmatic way to validate form controls.
```

I wonder if this should be less specific- it's not impossible that this could be added to, right? Maybe it could be just something like "An API with methods providing programatic access to HTML's form field constraint validations, including providing custom validations."?

---

Yeah, I think I'd prefer something like that too. @foolip ?

---

Sounds good to me!

---

Maybe we can preserve the search-ability here with something like this?

```suggestion
description: Methods that validate form controls before submission, such as `checkValidity()`, `reportValidity()` and `setCustomValidity()`.
```